hole/poker.go

```diff
@@ -5,7 +5,27 @@ import (
 	"slices"
 )
 
-func cardRune(number, suit int) rune {
+type CardValue int
+
+const (
+	ACE   CardValue = 0
+	TWO   CardValue = 1
+	THREE CardValue = 2
+	FOUR  CardValue = 3
+	FIVE  CardValue = 4
+	SIX   CardValue = 5
+	SEVEN CardValue = 6
+	EIGHT CardValue = 7
+	NINE  CardValue = 8
+	TEN   CardValue = 9
+	JACK  CardValue = 10
+	QUEEN CardValue = 11
+	KING  CardValue = 12
+)
```

Comment:
Also these are public, consts in Go don't do shouty or snakecase like in other languages, they just look like normal variables, so just name these "ace", "two", "three". Also we don't need to repeat ourselves so much, use iota and don't repeat the type link so:
```go
const (
    ace cardValue = iota + 1
    two
    three
    ...
)
```

Reply:
Or `1 + iota`, here's an example from the standard library - https://go.dev/src/time/time.go#L305

Reply:
![image](https://github.com/code-golf/code-golf/assets/1554858/d5b0e336-e2d9-4804-8e6b-9603b7c4574e)
tried with 1+ as well. i had forgotten this pattern, im unsure why it isnt working.